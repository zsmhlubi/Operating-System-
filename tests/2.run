./witsshell tests/2.in
