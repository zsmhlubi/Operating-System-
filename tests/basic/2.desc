Input to check ls command .. arg
