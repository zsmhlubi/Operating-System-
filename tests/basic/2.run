./witsshell tests/basic/2.in
