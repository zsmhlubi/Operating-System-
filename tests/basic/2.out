concurrency-mapreduce
concurrency-pzip
concurrency-webserver
concurrency-xv6-threads
filesystems-checker
filesystems-distributed
initial-kv
initial-reverse
initial-utilities
initial-xv6
INSTALL-xv6.md
processes-shell
README.md
scheduling-xv6-lottery
tester
vm-xv6-intro
